test/Interop/Cxx/templates/swift-class-instantiation-in-namespace-module-interface.swift

```diff
@@ -2,6 +2,9 @@
 // RUN: %target-swiftxx-frontend -emit-module -o %t/SwiftClassTemplateInNamespaceModule.swiftmodule %S/Inputs/SwiftClassTemplateInNamespaceModule.swift -I %S/Inputs -enable-library-evolution -swift-version 5
 // RUN: %target-swift-ide-test -print-module -module-to-print=SwiftClassTemplateInNamespaceModule -I %t/ -source-filename=x -enable-cxx-interop | %FileCheck %s
 
+// We need to be able to properly serialize the __ObjC module holding the "Space" enum.
+// REQUIRES: SR-XXXXX
```

Comment:
This test is failing because we can't find the enum decl because it's in the `__ObjC` module which isn't getting serialized. I think the best solution is to serialize that module as well and save it along side whatever the Swift module is that we're saving (`SwiftClassTemplateInNamespaceModule` in this case). But that's going to be a big/involved change, so I'd rather get this in first, and figure that out later. 

If we agree that this is the best path forward, I'll create an SR and update the comment.

Reply:
The contents of the `__ObjC` module should be generated by the importer, so I'm not yet convinced that serializing it is the right approach. I'd suggest to update the comment to say that there is a bug, and put the details into jira.

Reply:
OK good to know. I'll have to spend some more time thinking about this, then :) Here's the bug I filed: SR-14211.